code/game/objects/items/ego_weapons/zayin.dm

```diff
@@ -81,6 +271,31 @@
 	attack_verb_simple = list("slices", "slashes", "stabs")
 	hitsound = 'sound/weapons/bladeslice.ogg'
 
+	ability_cooldown_time = 30 SECONDS
+	matching_armor = /obj/item/clothing/suit/armor/ego_gear/letter_opener
+	use_message = "You use letter opener to send a message!"
+	use_sound = 'sound/items/handling/paper_drop.ogg'
+
+/obj/item/ego_weapon/support/letter_opener/Pulse(mob/user)
+	var/M = input(user,"Who would you like to message?","Select Someone") as null|anything in GLOB.player_list
```

Comment:
Needs to be refined as it can send messages to player ghosts.

Reply:
```
	var/M = input(user,"Who would you like to message?","Select Someone") as null|anything in sortList(GetPlayers())
```
```
/obj/item/ego_weapon/support/letter_opener/proc/GetPlayers()
	. = list()
	for(var/mob/living/carbon/human/H in GLOB.player_list)
		. += H
	return
```

Reply:
I added this and I believe it resolves the issue, but I don't know how to test things involving other players. It did work with a mob list, though.

Reply:
This did NOT work on its own, but it did make it very easy to add an isliving() check.